docs/framework/guides/contributing/development-environment.md

```diff
@@ -8,13 +8,17 @@ order: 10
 The CKEditor 5 codebase is divided into multiple [npm](http://npmjs.com/) packages, each developed in a separate Git repository. The main package is [`ckeditor5`](https://github.com/ckeditor/ckeditor5) which installs all project dependencies and various development-related resources such as:
 
 * the testing environment setup,
-* configuration for [Mr. Git](https://www.npmjs.com/package/mrgit) (a multi-repo management tool) and [Yarn](https://yarnpkg.com/) (a dependency management tool),
+* configuration for [Yarn](https://yarnpkg.com/),
 * translation management tools,
 * documentation generator,
 * and release tools.
 
 You can find all the official packages listed in [CKEditor 5 development repository's README](https://github.com/ckeditor/ckeditor5#packages).
 
+<info-box info>
+After the **19.0.0** release CKEditor 5 moved to the monorepo architecture. If you'd like to work with the older multi-repository release, please refer to [older Developer Environment guide](/docs/ckeditor5/19.0.0/framework/guides/contributing/development-environment.html) for multi-repository oriented instructions.
```

Comment:
The reader will be reading this from the perspective of CKEditor 5 being a monorepo already. So, what you want to explain here is that "prior to 19.0.0 CKEditor 5 was developed in multi repository architecture".

Reply:
Addressed in c2a07ff.